src/message/MessageActionSheet.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Clipboard } from 'react-native';
+import ActionSheet from 'react-native-actionsheet';
+
+import { narrowFromMessage } from '../utils/narrow';
+import { getSingleMessage } from '../api';
+
+const actionButtons = [
+  'Narrow to conversation',
+  'Reply',
```

Comment:
I removed `isFromReply` parameter - but right now Reply has the same functionality as Narrow to a conversation. That param had to make focus on text box when it'd be applicable